python/py-netifaces/Portfile

```diff
@@ -14,18 +14,17 @@ description         an easy way to get the address(es) of the machine’s networ
 long_description    ${description}
 
 homepage            https://alastairs-place.net/projects/netifaces/
-master_sites        ${homepage}
-
 distname            netifaces-${version}
```

Comment:
you should be able to remove the `distname` field as well, this is the default set by the `python` PG

Reply:
Done, thanks for the review.